packages/waggle/src/socket/IOProxy.ts

```diff
@@ -44,12 +44,7 @@ export default class IOProxy {
 
   public subscribeForTopic = async (peerId: string, channelData: IChannelInfo) => {
     log(`${peerId} is subscribing for channel ${channelData.address}`)
```

Comment:
should be "is subscribing **to** channel $..."

Reply:
corrected